packages/webpack-config-terra/lib/postcss/ThemePlugin.js

```diff
@@ -0,0 +1,51 @@
+const postcss = require('postcss');
+const removeCssModulesPseudoClasses = require('./_removeCssModulesPseudoClasses');
+const getThemeConfig = require('../utils/_getThemeConfig');
+
+const SUPPORTED_THEMES = [
```

Comment:
Can we list these in the doc?

Reply:
i added them to the terra-theme.config.js docs